docs/agent/config/v3.mdx

```diff
@@ -0,0 +1,584 @@
+
+# ----------------------------------------------------------------------
+# | Log Format                                                         |
+# ----------------------------------------------------------------------
+
+# Specifies the format of log records.
+#
+# Allowed values:
+#   logfmt  - Human and machine-friendly key/value pairs.
+#     json  - Newline-separated JSON objects.
+#     term  - Colored format for TTY; otherwise, same as logfmt.
+
+# log_format: json
+
+# ----------------------------------------------------------------------
+# | Log                                                                |
+# ----------------------------------------------------------------------
+
+# This is the destination where ngrok should write the logs.
+#
+# Allowed values:
+#   stdout  - write to standard out
+#   stderr  - write to standard error
+#    false  - disable logging
+#   <path>  - write log records to file path on disk
+
+# log: /var/log/ngrok.log
+
+# ----------------------------------------------------------------------
+# | Metadata                                                           |
+# ----------------------------------------------------------------------
+
+# Custom string included in the ngrok API response for endpoints.
+#
+# Maximum 4096 characters.
+
+# metadata: '{"serial": "00012xa-33rUtz9", "comment": "For customer alan@example.com"}'
+
+# ----------------------------------------------------------------------
+# | Proxy URL                                                          |
+# ----------------------------------------------------------------------
+
+# URL of an HTTP or SOCKS5 proxy for endpoint connections.
+#
+#   (!) ngrok will also respect the `http_proxy` and `http_proxy_env`
+#       environment variables.
+
+# proxy_url: socks5://localhost:9150
+
+# ----------------------------------------------------------------------
+# | Remote Management                                                  |
+# ----------------------------------------------------------------------
+
+# Allows remote management of the ngrok agent (stop, restart, update) via
+# the ngrok API or Dashboard.
+#
+# Defaults to `true`.
+
+# remote_management: false
+
+# ----------------------------------------------------------------------
+# | Root CAs                                                           |
+# ----------------------------------------------------------------------
+
+# Root certificate authorities used to validate TLS connections to the
+# ngrok server.
+#
+# Allowed values:
+#    trusted - Use only the trusted certificate root for ngrok.com.
+#       host - Use the root certificates trusted by the host's OS (useful for MITM proxies with DPI).
+#     <path> - Path to a PEM file with additional trusted certificate authorities.
+
+# root_cas: trusted
+
+# ----------------------------------------------------------------------
+# | Server Address                                                     |
```

Comment:
i think agent.connect_address replaces this in v3

Reply:
😓  @warrn Do you happen to know off the top of your dome on this one?

Reply:
Let me update this 👍